reverse_engineering/helpers/utils.js

```diff
@@ -98,6 +98,42 @@ const getTemplateDocByJsonSchema = schema => {
 	}, {});
 };
 
+/**
+ *
+ * @param {Array<number | string | boolean>} list
+ * @returns {number | string | boolean}
+ */
+const getMostFrequentValueInList = list => {
+	const itemsToFrequencyMap = {};
```

Comment:
it's one-to-one map (not many-to-one) so better to call it - `itemToFrequencyMap`